test/mysql/index.js

```diff
@@ -140,6 +140,19 @@ describe('with MySQL client', () => {
       sinon.restore(ModelPrototype);
     });
 
+    it('should not stringify \'\' values on update with `patch` option', async () => {
```

Comment:
This test does not assert that the value has not been parsed, change to the following:

```js
it('should keep an empty string on update with `patch` option', async () => {
  const model = await Model.forge({ foo: 'bar' }).save();

  await model.save({ foo: '' }, { patch: true });

  model.get('foo').should.equal('');
});
```